### PR TITLE
Added new questions to general-cs-quiz.ts

### PR DESCRIPTION
**Pull request by Atharva-Morankar-09**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,24 @@
 const generalCSQuiz = [
+  {
+    Question: "If you forget one semicolon or misplace a character, the entire program will not run and send you back what error?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:3]:
```suggestion
    Question: "What is the name for an error where the programmer writes invalid syntax resulting in the program failing to run?",
```

File: src/data/general-cs-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+    Answer: "Syntax Error",
+    Distractor1: "Runtime Error",
+    Distractor2: "Logical Error",
+    Distractor3: "Semantic Error",
+    Explanation: "A syntax error occurs when the programmer writes an instruction using incorrect syntax. ",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:8]: Please avoid using direct quotes from 3rd party resources like this one
https://www.sciencedirect.com/topics/engineering/syntax-error

```suggestion
    Explanation: "A syntax error is when the programmer writes invalid code that violates the rules of that programming language.",
```

File: src/data/general-cs-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+    Link: "https://www.youtube.com/watch?v=zOjov-2OZ0E",
+  },
+  {
+    Question:
+      "Which operation is used to add an item to the queue?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:13]:
```suggestion
      "Which operation is used to add an item to the back of the queue?",
```

File: src/data/general-cs-quiz.ts [hunk 4]
```diff
@@ -1,4 +1,24 @@
...
+    Answer: "Enqueue",
+    Distractor1: "Dequeue",
+    Distractor2: "Peek",
+    Distractor3: "isFull",
+    Explanation:
+      "Enqueue is the operation of inserting elements in the queue.",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:19]:
```suggestion
      "Enqueue is the operation is used to add an item to the back of the queue.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Atharva-Morankar-09** [comment]
Hey, I have done the required changes. Thank you.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**